openrtb_ext/imp_onetag.go

```diff
@@ -0,0 +1,10 @@
+package openrtb_ext
+
+import (
+	"encoding/json"
+)
+
+type ExtImpOnetag struct {
+	PubId string          `json:"pubId"`
```

Comment:
OpenRTB already includes a publisher id field. Why do you need to use a separate one?

Reply:
This is intended as an Account ID, which uniquely identifies a Publisher on our system and it has a different meaning from the request.{site,app}.publisher.id